mmv1/templates/terraform/examples/identity_platform_project_default_config.tf.erb

```diff
@@ -18,4 +18,30 @@ resource "google_identity_platform_project_default_config" "<%= ctx[:primary_res
             }
         }
     }
-}
\ No newline at end of file
+
+    blocking_functions {
+        triggers {
+            "beforeSignIn" = {
+                function_uri = "new_uri-before-sign-in"
+            }
+        }
+
+        forward_inbound_credentials {
+            refresh_token = true
+            access_token = true
+            id_token = true
+        }
+    }
+
+    quota {
+        sign_up_quota_config {
+            quota = 1000
+        }
+    }
+
+    authorized_domains = [
+        "localhost",
+        "project_id.firebaseapp.com",
```

Comment:
I was thinking more of doing it properly, like `"<%= ctx[:vars]['project_id'] %>"`, or actually put in a [data source](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/data-sources/project) and reference.